lms/djangoapps/student_account/views.py

```diff
@@ -293,3 +303,63 @@ def _external_auth_intercept(request, mode):
         return external_auth_login(request)
     elif mode == "register":
         return external_auth_register(request)
+
+
+@login_required
+@require_http_methods(['GET'])
+def account_settings(request):
+    """Render the students account settings page.
+
+    Args:
+        request (HttpRequest)
+
+    Returns:
+        HttpResponse: 200 if the page was sent successfully
+        HttpResponse: 302 if not logged in (redirect to login page)
+        HttpResponse: 405 if using an unsupported HTTP method
+
+    Example usage:
+
+        GET /account/settings
+
+    """
+    released_languages = DarkLangConfig.current().released_languages_list
```

Comment:
It seems strange to me that this code goes through DarkLangConfig to access the released list of languages, but I see that's how the existing code works too. I would think that DarkLangConfig was only for dealing with languages that have been released 'dark'.